qml/ThemeColors.qml

```diff
@@ -0,0 +1,42 @@
+/*
+ * <one line to give the program's name and a brief idea of what it does.>
+ * Copyright (C) 2018  <copyright holder> <email>
+ * 
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ * 
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ * 
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import QtQuick 2.0
+
+/**
+ * This Component is used to specify colors that depends on a theme but cannot be suppliy by it.
+ */
+
+QtObject {
+    property var themes : {
+		'Ubuntu.Components.Themes.Ambiance' : {
```

Comment:
use 4 spaces for indentation, not tabs :-)

Reply:
these time it wasn't me ;-P -> done